utils/hare-fetch-fids

```diff
@@ -198,7 +291,13 @@ def get_nodes_for_cluster(path: str) -> List[Node]:
             break
 
     keys = get_keys_for_processes(file_data, client_types)
-    process_all_keys(keys, nodes, client_types)
+
+    Node.profile_fid = get_profile_fid(file_data)
+    for node in nodes:
+        node.ha_ep = get_ha_ep(file_data, node.name)
+        node.uuid = generate_node_uuid()
```

Comment:
I think we should start using `machine-id` and store it in Consul and use that as `node.uuid` instead of generating this by ourselves. May be this can be done in separate task.